build.gradle

```diff
@@ -2861,6 +2861,7 @@ project(':streams:integration-tests') {
   dependencies {
     implementation libs.slf4jApi
 
+    api libs.scalaLibrary
```

Comment:
Why `api` versus `implementation`?

Reply:
[Gradle Document](https://docs.gradle.org/current/samples/sample_building_scala_libraries.html)

The api means  "This dependency is exported to consumers, that is to say found on their compile classpath", thus I think configure `api` is enough to resolve the problem.

Reply:
`api` is when you want to export the dependency while `implementation` is when you do _not_ want to export the dependency. I think we want the latter here. We don't return Scala library types from the public API of this module, do we?

Reply:
You are right, I will fix it